lib/src/main/java/com/grio/lib/features/BugSnap.kt

```diff
@@ -134,6 +134,7 @@ class BugSnap {
                                 override fun onRecordingFinished(file: File) {
                                     val intent = Intent(context, ReporterActivity::class.java)
                                     intent.putExtra("videoFile", file)
+                                    intent.putExtra("reportType", "video")
```

Comment:
I recommend having the ReporterActivity class have a newIntent method where it constructs the intent to start itself. This simplifies other activities not needing to know what extra fields it needs to properly start.

I have an example here: https://github.com/GrioSF/bugsnap-android/blob/17c607c539ee6e1f1a213afc9cafd9118c38ef46/lib/src/main/java/com/grio/lib/features/editor/EditorActivity.kt#L50